copyright/README.md

```diff
@@ -0,0 +1,30 @@
+## Mobify Copyright Headers
+
+This tool reads from the `headers/copyright.extension.txt` text files to select what header to insert, where `extension` is the target file extension
+
+## Quick Start
+
+- Clone this repo
```

Comment:
I've seen these formatted as a short terminal session... sometimes that's clearer.  Eg.

```sh
git clone git@github.com:mobify/mobify-code-style.git
cd mobify-code-style/copyright
node copyright.js ../**/*.js ../**/*.py
```